active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+# Background
+
+In many applications, media and data is shown in various layouts like lists, grids, and tables.
+Instead of scrolling or panning through the content,
+users are able to page through the content with previous and next buttons or jump to specific pages. 
+
+This UI pagination behavior is supported in WinUI currently with the PagerControl. 
+(See PagerControl [spec](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/savoyschuler/pagercontrol/active/PagerControl/PagerControl.md) 
+and PagerControl [proposal](https://github.com/microsoft/microsoft-ui-xaml/issues/60) 
+for the latest updates.) The PagerControl works with page numbers in a few different formats:
+
+![PagerControl example](images/PagerControl.jpg)
+
+A common UI pattern not yet supported by the existing PagerControl visualizations appears in
+media viewer scenarios like photo carousels or app lists.
+In these scenarios, the pages are commonly indicated with 'dot' glyphs instead of
+numbers to encourage discovery of content and reduce unneccessary UI elements on the page.
+Conceptually, this is similar to an unordered list represented by bullet points instead of numbers. 
+
+![A sketch of five horizontal dots with the first selected and the previous and next buttons are visible](images/PipsPager_Visible.gif)
+
+The new PipsPager control can  be used when the page content is not explicitly ordered and
+a symbol or glyph can instead represent the page number.
+
+# API Pages
+
+## PipsPager class
+
+Represents a control that provides interaction and navigation of a paginated layout view 
+(ListView, GridView, ItemsRepeater, DataGrid, etc.) when the page numbers do not need to be visually known.
+
+The PipsPager, like the other pager controls, is independent of the layout views and
+does not control any of the data being displayed in the layout view.
+
+### Is this the right control?
+
+Use a PipsPager when the content in the layout is not explicitly ordered by relevancy or 
```

Comment:
It's still totally ordered. I think the notion of unordered here is because it's like a bulleted list, and in HTML a bulleted list is called an "unordered list". But I think this just makes the description confusing.

Reply:
Perhaps "not explicitly *numbered*, or you desire a glyph-based representation".

Reply:
Recommend